<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminwindow</class>
 <widget class="QMainWindow" name="adminwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>点都德++餐厅服务与管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabBar::tab{
background: rgb(188, 203, 255);
	color: rgb(85, 85, 127);
	padding:20px;
width:200px;
height:30px;
	font: 14pt &quot;微软雅黑&quot;;
}
QTabBar::tab:selected{
background-color: rgb(189, 232, 255);

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>700</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>菜单列表</string>
     </attribute>
     <widget class="QTableWidget" name="mTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>1160</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 菜品名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ldishQuery">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="bdishQuery1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>7</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bdishShow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示全部</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cQuery">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>7</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>蒸包类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>煎焗类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>蔬菜类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>肠粉类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>饭粥类</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>甜品类</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 菜品种类：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bdishQuery2">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>7</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>按类查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bdishEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>520</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>编辑菜品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bdishAdd">
      <property name="geometry">
       <rect>
        <x>665</x>
        <y>520</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>增加菜品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bdishDel">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>520</y>
        <width>180</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除菜品</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string> 顾客列表</string>
     </attribute>
     <widget class="QTableWidget" name="cusTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>1160</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="bcusDel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>520</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton::text{
}
color: rgb(187, 226, 255);</string>
      </property>
      <property name="text">
       <string>删除无效账号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>8</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>顾客手机号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lcusQuery">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="bcusQuery">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>8</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bcusShow">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>8</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示全部</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>   厨师列表</string>
     </attribute>
     <widget class="QTableWidget" name="cookTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>1160</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>170</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>厨师工作证号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lcookQuery">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="bcookQuery">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>8</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bcookShow">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>8</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示所有</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bcookDel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>520</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton::text{
}
color: rgb(187, 226, 255);</string>
      </property>
      <property name="text">
       <string>删除无效账号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bcookAdd">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>520</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton::text{
}
color: rgb(187, 226, 255);</string>
      </property>
      <property name="text">
       <string>添加新账号</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string> 服务员列表</string>
     </attribute>
     <widget class="QTableWidget" name="wTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>1160</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  服务员工作证号：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bwaiterQuery">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>8</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bwaiterShow">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>8</y>
        <width>150</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示所有</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lwaiterQuery">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="bwaiterAdd">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>520</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton::text{
}
color: rgb(187, 226, 255);</string>
      </property>
      <property name="text">
       <string>添加新账号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bwaiterDel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>520</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton::text{
}
color: rgb(187, 226, 255);</string>
      </property>
      <property name="text">
       <string>删除无效账号</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>其他</string>
     </attribute>
     <widget class="QTableWidget" name="tTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>400</width>
        <height>450</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>11</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 显示当前所有餐桌号：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btableAdd">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加桌号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btableDel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>530</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除桌号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>80</y>
        <width>600</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    点都德++餐厅服务与管理系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>140</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C++程序设计与训练</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>200</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 设计者</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>260</y>
        <width>400</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>钟清扬     2016011481</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>300</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>zhongqy16@mails.tsinghua.edu.cn</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="bLogout">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>425</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bExit">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>425</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
